lib/builtin.pm

```diff
@@ -168,6 +168,31 @@ numerical argument.
 Returns the largest integer value less than or equal to the given numerical
 argument.
 
+=head2 indexed
+
+    @kvlist = indexed(@list)
```

Comment:
Calling an @array a "list" just propagates the list/array confusion. This should not use the word list on either side.

   @iv = indexed(@array);
   @iv = indexed("foo","bar","baz","bop");
   
 would be better IMO. Or even
 
   @idx_val = indexed(@array);
   @idx_val = indexed("foo","bar","baz","bop");
  
since this is a doc.